src/droplet.F

```diff
@@ -94,13 +93,13 @@ subroutine droplet_driver(dt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,
       real, intent(in), dimension(kb:ke+1) :: sigmaf
       real, intent(in), dimension(ib:ie,jb:je) :: znt
       real, intent(in), dimension(ibl:iel,jbl:jel) :: s10
-      real, intent(inout), dimension(ib:ie,jb:je,kb:ke) :: rho,prs
+      real, intent(inout), dimension(ib:ie,jb:je,kb:ke) :: rho,prs,ta
       real, intent(inout), dimension(ib:ie,jb:je,kb:ke) :: pi0,th0,th_in,ppa
-      real, intent(inout), dimension(ib:ie,jb:je,kb:ke) :: thten,qten
+      double precision, intent(inout), dimension(ib:ie,jb:je,kb:ke) :: uten,vten,wten,thten,qten,dten
```

Comment:
Again the potentially inconsistent dimension between `dten`, `uten`, `vten` and `wten`.

Reply:
See above